plugins/copr.py

```diff
@@ -482,9 +483,19 @@ def _download_repo(self, project_name, repo_filename):
                     import urllib
                     res = urllib.urlopen(self.copr_url + "/coprs/" + project_name)
                     status_code = res.getcode()
-                if str(status_code) != '404':
-                    raise dnf.exceptions.Error(_("This repository does not have"
-                                                 " any builds yet so you cannot enable it now."))
+                if status_code != 404:
+                    error_msg = "It wasn't possible to enable this project.\n"
+                    try:
+                        if PY3:
+                            response = urllib.request.urlopen(self.copr_url + api_path)
+                        else:
+                            response = urllib.urlopen(self.copr_url + api_path)
```

Comment:
I think we already have the response in 'res' variable - so it shouldn't be needed to urlopen the file again.

Reply:
Aha, I see...  can we drop the `urllib.urlopen(self.copr_url + "/coprs/" + project_name)` calls then?  I don't think they are useful anymore, as long as we only provide message like `Such repository does not exist.` (and that could be enhanced a bit, like "Copr project `aaa/bbb` doesn't exist".

Reply:
If we drop those calls we will not know if the `dnf copr enable` fails because the project doesn't exist or just the chroot doesn't exist.

Reply:
Perhaps we could blindly claim that project doesn't exist if there's no error message?  Or does that sound too non-deterministic/optimistic :-P

Reply:
I would rather not make any estimates  :slightly_smiling_face: